### ISSUE TITLE
Retire the 'raw' orography directory

### ISSUE THREAD

**Issue by GeorgeGayno-NOAA** [NOAA-EMC/UFS_UTILS#984]
UFS_UTILS currently points to four 'fixed' directories (for example, on Hera):

```
am -> /scratch1/NCEPDEV/global/glopara/fix/am/20220805
orog -> /scratch1/NCEPDEV/global/glopara/fix/orog/20231027
orog_raw -> /scratch1/NCEPDEV/global/glopara/fix/raw/orog
sfc_climo -> /scratch1/NCEPDEV/global/glopara/fix/sfc_climo/20230925
```

The 'orog_raw' directory will be removed. Any needed files from 'orog_raw' will be pulled into 'orog'.

UFS_UTILS will need some minor updates for this new directory structure.

**Comment by GeorgeGayno-NOAA** [comment]
@KateFriedman-NOAA has setup test directories on Hera: `/scratch1/NCEPDEV/global/glopara/TEST_FIX`

**Comment by GeorgeGayno-NOAA** [comment]
@KateFriedman-NOAA - UFS_UTILS no longer uses these files:

- gmted2010.30sec.int
- landcover30.fixed
- thirty.second.antarctic.new.bin

It now uses the NetCDF versions:

- topography.gmted2010.30s.nc
- landcover.umd.30s.nc
- topography.antarctica.ramp.30s.nc

**Comment by GeorgeGayno-NOAA** [comment]
Looking through the code and scripts that add lakes to our grids, the following files are needed:

GlobalLakeStatus:

- GlobalLakeStatus_GLDBv3release.dat
- GlobalLakeStatus_MODISp.dat
- GlobalLakeStatus_VIIRS.dat

GlobalLakeDepth

- GlobalLakeDepth_GLDBv3release.dat
- GlobalLakeDepth_GLOBathy.dat

All other 'Status' and 'Depth' files don't appear to be used.

**Comment by KateFriedman-NOAA** [comment]
> @KateFriedman-NOAA - UFS_UTILS no longer uses these files:
> 
> * gmted2010.30sec.int
> * landcover30.fixed
> * thirty.second.antarctic.new.bin
> 
> It now uses the NetCDF versions:
> 
> * topography.gmted2010.30s.nc
> * landcover.umd.30s.nc
> * topography.antarctica.ramp.30s.nc

@GeorgeGayno-NOAA Noted. UFS_UTILS sources the NetCDF versions from the `raw/orog` folder, right?

> Looking through the code and scripts that add lakes to our grids, the following files are needed:
> 
> GlobalLakeStatus:
> 
> * GlobalLakeStatus_GLDBv3release.dat
> * GlobalLakeStatus_MODISp.dat
> * GlobalLakeStatus_VIIRS.dat
> 
> GlobalLakeDepth
> 
> * GlobalLakeDepth_GLDBv3release.dat
> * GlobalLakeDepth_GLOBathy.dat
> 
> All other 'Status' and 'Depth' files don't appear to be used.

Also noted. Given the lists you just provided and once you confirm the NetCDF versions are coming from `raw/orog`...I will build a new `orog` timestamp. In this new timestamp copy I will pull in the files that we need to retain from the `raw/orog` folder.

Before removing the `raw/orog` folder I will also need to pull copies through the symlinks into the older `orog` timestamps so we don't break things for folks still using the older `orog` timestamps. Will get us moved to the new `orog` timestamp before proceeding with the removal and such.

Any concerns or questions about the above plan?

**Comment by KateFriedman-NOAA** [comment]
FYI @WalterKolczynski-NOAA

**Comment by GeorgeGayno-NOAA** [comment]
> > @KateFriedman-NOAA - UFS_UTILS no longer uses these files:
> > 
> > * gmted2010.30sec.int
> > * landcover30.fixed
> > * thirty.second.antarctic.new.bin
> > 
> > It now uses the NetCDF versions:
> > 
> > * topography.gmted2010.30s.nc
> > * landcover.umd.30s.nc
> > * topography.antarctica.ramp.30s.nc
> 
> @GeorgeGayno-NOAA Noted. UFS_UTILS sources the NetCDF versions from the `raw/orog` folder, right?

Yes. 

> 
> > Looking through the code and scripts that add lakes to our grids, the following files are needed:
> > GlobalLakeStatus:
> > 
> > * GlobalLakeStatus_GLDBv3release.dat
> > * GlobalLakeStatus_MODISp.dat
> > * GlobalLakeStatus_VIIRS.dat
> > 
> > GlobalLakeDepth
> > 
> > * GlobalLakeDepth_GLDBv3release.dat
> > * GlobalLakeDepth_GLOBathy.dat
> > 
> > All other 'Status' and 'Depth' files don't appear to be used.
> 
> Also noted. Given the lists you just provided and once you confirm the NetCDF versions are coming from `raw/orog`...I will build a new `orog` timestamp. In this new timestamp copy I will pull in the files that we need to retain from the `raw/orog` folder.
> 
> Before removing the `raw/orog` folder I will also need to pull copies through the symlinks into the older `orog` timestamps so we don't break things for folks still using the older `orog` timestamps. Will get us moved to the new `orog` timestamp before proceeding with the removal and such.
> 
> Any concerns or questions about the above plan?

Sounds good. I have tests ready to go.

**Comment by KateFriedman-NOAA** [comment]
@GeorgeGayno-NOAA Based on the info above I built a new timestamp orog folder, see it here on Hera: `/scratch1/NCEPDEV/global/glopara/fix/orog/20240917`

Let me know if that works for testing and if there is anything missing or can be further removed. Thanks!

**Comment by GeorgeGayno-NOAA** [comment]
Using dcd2fd9 on Hera, the 'fixed' directories were set up with `./link_fixdirs.sh` 

```
/scratch1/NCEPDEV/da/George.Gayno/ufs_utils.git/UFS_UTILS/fix $ ls -l
total 8
lrwxrwxrwx 1 George.Gayno da   48 Sep 18 15:06 am -> /scratch1/NCEPDEV/global/glopara/fix/am/20220805
-rwxr-xr-x 1 George.Gayno da 2094 Sep 18 14:58 link_fixdirs.sh
lrwxrwxrwx 1 George.Gayno da   50 Sep 18 15:06 orog -> /scratch1/NCEPDEV/global/glopara/fix/orog/20240917
drwxr-sr-x 2 George.Gayno da 4096 Sep 13 19:09 orog.test
lrwxrwxrwx 1 George.Gayno da   55 Sep 18 15:06 sfc_climo -> /scratch1/NCEPDEV/global/glopara/fix/sfc_climo/20230925
```
Then, the grid_gen regression tests were run. All passed as expected:

```
/scratch1/NCEPDEV/da/George.Gayno/ufs_utils.git/UFS_UTILS/reg_tests/grid_gen $ more summary.log
consistency.log01:<<< C96 UNIFORM TEST PASSED. >>>
consistency.log02:<<< C96 VIIRS BNU TEST PASSED. >>>
consistency.log03:<<< GFDL REGIONAL TEST PASSED. >>>
consistency.log04:<<< ESG REGIONAL TEST PASSED. >>>
consistency.log05:<<< ESG REGIONAL PERCENT CATEGORY TEST PASSED. >>>
consistency.log06:<<< REGIONAL 12 THREAD GSL GWD TEST PASSED. >>>
consistency.log07:<<< REGIONAL 24 THREAD GSL GWD TEST PASSED. >>>
```

**Comment by GeorgeGayno-NOAA** [comment]
The `./driver_scripts/driver_grid.hera.sh` was run on Hera using dcd2fd9. The lake code was turned on and all combinations of lake status/lake depth choices were tested. In all five tests were run by adjusting the script's 'lake_data_srce' variable as follows:

```
# choose dataset sources for lakefrac & lakedepth so that lake_data_srce=LakeFrac_LakeDepth;
# available options are 'MODISP_GLDBV3', 'MODISP_GLOBATHY', 'VIIRS_GLDBV3', 'VIIRS_GLOBATHY' & 'GLDBV3'
# Test 1
export lake_data_srce=MODISP_GLDBV3
# test 2
#export lake_data_srce=MODISP_GLOBATHY
# test 3
#export lake_data_srce=VIIRS_GLDBV3
# test 4
#export lake_data_srce=VIIRS_GLOBATHY
# test 5
#export lake_data_srce=GLDBV3
```

Test 1- used "GlobalLakeStatus_MODISp.dat" and "GlobalLakeDepth_GLDBv3release.dat"
Test 2- used "GlobalLakeStatus_MODISp.dat" and "GlobalLakeDepth_GLOBathy.dat"
Test 3- used "GlobalLakeStatus_VIIRS.dat" and "GlobalLakeDepth_GLDBv3release.dat"
Test 4- used "GlobalLakeStatus_VIIRS.dat" and "GlobalLakeDepth_GLOBathy.dat"
Test 5- used "GlobalLakeStatus_GLDBv3release.dat" and "GlobalLakeDepth_GLDBv3release.dat"

All tests ran to completion.

**Comment by GeorgeGayno-NOAA** [comment]
All regression tests were run on Hera using dcd2fd9. All passed as expected.

**Comment by GeorgeGayno-NOAA** [comment]
@KateFriedman-NOAA - all files needed by UFS_UTILS are there. If you replicate this new directory structure on all machines, I can open a PR.

**Comment by KateFriedman-NOAA** [comment]
Awesome...we'll sync to the other platforms and confirm when that's done.

@WalterKolczynski-NOAA please sync the new `orog/20240917` to MSU, thanks!

**Comment by bubba0077** [comment]
Synchronized to MSU

**Comment by WalterKolczynski-NOAA** [comment]
Synchronized to MSU

**Comment by KateFriedman-NOAA** [comment]
Synced to WCOSS2, Jet, and AWS.

**Comment by GeorgeGayno-NOAA** [comment]
> Synced to WCOSS2, Jet, and AWS.

Where on AWS? https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/fix/fix_orog/

**Comment by KateFriedman-NOAA** [comment]
> Where on AWS?

Here: https://noaa-nws-global-pds.s3.amazonaws.com/index.html#fix/orog/20240917/
